compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUCheckIRBeforeLLVMConversion.cpp

```diff
@@ -62,7 +62,8 @@ void LLVMCPUCheckIRBeforeLLVMConversionPass::runOnOperation() {
     return signalPassFailure();
   }
   int maxAllocationSizeInBits = clMaxAllocationSizeInBytes * 8;
-  if (totalBits > maxAllocationSizeInBits) {
+  if (clFailUnboundDynamicStackAllocation &&
```

Comment:
This option name doesn't seem to mean skip static bound checking. Maybe add a new option for that or change its name?